template.yaml

```diff
@@ -1291,9 +1293,14 @@ Resources:
         Variables:
           AWC_ACCOUNT_ID: !Ref AWS::AccountId
           TABLE_NAME: !Ref NvaResourcesTable
-          EVENT_BUS: !Ref CristinImportEventBusName
+          EVENT_BUS: !GetAtt ImportRequestsEventBus.Name
 
 
+  #=============================Event Buses =============================================================
+  ImportRequestsEventBus:
+    Type: AWS::Events::EventBus
+    Properties:
+      Name: !Join [ '',[ 'import-requests-event-bus',!Ref Suffix ] ]
```

Comment:
The Suffix allows the creation of a separate event-bus for every feature branch when the new pipeline will be in place